src/xpcc/architecture/platform/driver/adc/stm32f0/adc.hpp.in

```diff
@@ -0,0 +1,263 @@
+// coding: utf-8
+/*
+ * Copyright (c) 2018, Álan Crístoffer
+ *
+ * This file is part of the xpcc project.
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+// ----------------------------------------------------------------------------
+
+#ifndef XPCC_STM32F0_ADC_HPP
+#define XPCC_STM32F0_ADC_HPP
+
+#include <stdint.h>
+#include "../../../type_ids.hpp"
+#include "../../../device.hpp"
+#include <xpcc/architecture/interface/register.hpp>
+
+namespace xpcc
+{
+namespace stm32
+{
+/**
+ * Analog/Digital-Converter module (ADC1).
+ *
+ * The 12-bit ADC is a successive approximation analog-to-digital
+ * converter. It has up to 19 multiplexed channels allowing it measure
+ * signals from 16 external and three internal sources.
+ * The result of the ADC is stored in a left-aligned or right-aligned
+ * 16-bit data register.
+ *
+ * This API is designed for the internal ADCs of STM32F0x1/STM32F0x2/STM32F0x8
+ *
+ * \author	Álan Crístoffer
+ * \ingroup	stm32
```

Comment:
Could you define and add the doxygen group like [here](https://github.com/roboterclubaachen/xpcc/blob/develop/src/xpcc/architecture/platform/driver/adc/stm32/adc.hpp.in#L43-L46) and [here](https://github.com/roboterclubaachen/xpcc/blob/develop/src/xpcc/architecture/platform/driver/adc/stm32/adc.hpp.in#L66)?